kolibri/core/sqlite/management/commands/vacuumsqlite.py

```diff
@@ -0,0 +1,49 @@
+import logging as logger
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db import DEFAULT_DB_ALIAS
+
+logging = logger.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Vacuum Kolibri's SQLite database to optimize it and reduce the .wal file size"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--database', action='store', dest='database', default=DEFAULT_DB_ALIAS,
+            help='Specifies the database to vacuum. Defaults to the "default" database.',
+        )
+        parser.add_argument(
+            '--interval', action='store', dest='interval', default=0, type=int,
+            help='Specifies the interval (in minutes) to run the process continuosly. If 0, no repetition will happen',
+        )
+
+    def handle(self, *args, **options):
+        database = options['database']
+        connection = connections[database]
+        interval = options['interval']
+        if connection.vendor == "sqlite":
+            while True:
+                self.perform_vacuum(connection)
+                if not interval:
+                    break
+                logging.info("Next Vacuum in {interval} minutes.".format(interval=interval))
```

Comment:
`debug` level this.

Reply:
why? other similar commands in Kolibri use info for these things (look a the ping command)

Reply:
I am just trying to limit our default level user logs. If you `debug` level it, it will be visible with `kolibri <command> --debug`

Reply:
@benjaoming yes, I know it and I don't dislike your suggestion. My only objection is the same I had for the logging part: I'd prefer to keep the consistency with other similar cases.